honeybee_radiance_folder/gridutil.py

```diff
@@ -224,3 +227,54 @@ def restore_original_distribution(
                     # write the lines to the output file
                     for _ in range(end - st + 1):
                         outf.write(next(srf))
+
+
+def parse_grid_info(
+        info_file, validate=True,
+        validation_keys=('name', 'identifier', 'count', 'group', 'full_id')):
+    """Parse sensor grid information from a info json file. This information
+    typically contains full_id, name, identifier, group and count. In the case
+    of _model_grids_info json, it will also contain the starting line (start_ln)
+
+    Args:
+        info_file: Path to the grid info file.
+        validate: If set to True, check if the grid_info contains all the
+            necessary keys to define the grid. The necessary keys are specified
+            through the input validation_keys.
+        validation_keys: The minimum subset of dict keys that each constituent
+        dictionary in the grid_info list should have.
+
+    Returns:
+        A list containing the information about sensor grids.
+    """
+    if not os.path.isfile(info_file):
+        return []
+
+    with open(info_file) as inf:
+        grid_info = json.load(inf)
+
+    if validate and grid_info:
+        for grid_info_dict in grid_info:
+            grid_info_keys = grid_info_dict.keys()
+            for key in validation_keys:
+                assert key in grid_info_keys, \
+                    'Did not find the key %s in the grid_info parsed from %s' % (
+                        key, info_file)
```

Comment:
I don't think we need an extra validation here. It's too generic. I would allow the downstream methods to do the validation for what they need from the info.

Reply:
Alright.